src/logger.ts

```diff
@@ -0,0 +1,28 @@
+import * as pino from 'pino';
+import { TextChannel } from 'discord.js';
+
+import { client } from './client';
+import config from './config';
+
+const logger = pino({
+    prettyPrint: { colorize: true }
+});
```

Comment:
Maybe it would be useful to configure the log level for pino? If we added a `debug` function to the `log` object, [it wouldn't appear](https://github.com/pinojs/pino/blob/afb639c79a1135790e9302b94f054a6118bffa6f/docs/api.md#bindingslevel-string) because the default level is 'info'. Also, the service we use to deploy might restrict too many log lines.

This is just a suggestion since I don't know the limits of logging where the bot is deployed.
```suggestion
const logger = pino({
    prettyPrint: { colorize: true },
    level: process.env.LOG_LEVEL || 'info'
});
```

Reply:
Good idea, this will help reduce the noise if needed 👍